README.md

```diff
@@ -711,6 +711,34 @@ You can also deploy it to Cloudflare Pages.
 wrangler pages deploy ./dist
 ```
 
+## Development
+
+### Wrangler Bindings and Variables
+
+If you use a `wrangler.toml` file to declare Bindings (Such as a D1 Database) and Variables , these can be used in your HonoX application during local development (with the `vite` command) with a small change to your vite.config.ts file.
+
+Import the `getPlatformProxy` function from the `wrangler` package and use it to get the environment variables and dispose function. Then, pass the environment variables to your HonoX application devServer configuration.
+
+```ts
+// vite.config.ts
+import { defineConfig } from 'vite'
+import { getPlatformProxy } from 'wrangler'
+
+export default defineConfig(async () => {
+  const { env, dispose } = await getPlatformProxy()
+  return {
+    plugins: [
+      honox({
+        devServer: {
```

Comment:
Since [this fix](https://github.com/honojs/vite-plugins/pull/90) made the type correct, it would be easier to specify "`env` and `onServerClose`" as one plugin:

```ts
export default defineConfig(async () => {
  const { env, dispose } = await getPlatformProxy()
  return {
    plugins: [
      honox({
        devServer: {
          plugins: [
            {
              env,
              onServerClose: dispose
            }
          ]
        }
      })
    ]
  }
})
```

Reply:
oo, nice. Will update to match :D